model/events/LtiAgsListener.php

```diff
@@ -117,6 +120,7 @@ private function onDeliveryExecutionFinish(DeliveryExecutionState $event): void
             /** @var QueueDispatcherInterface $taskQueue */
             $taskQueue = $this->getServiceLocator()->get(QueueDispatcherInterface::SERVICE_ID);
             $taskQueue->createTask(new SendAgsScoreTask(), [
+                SendAgsScoreTask::RETRY_MAX => $this->getAgsMaxRetries(),
```

Comment:
For consistency, you could move all parameter names to the constant or write only the name and do not use the constant